### PR TITLE
project structure

### PR DESCRIPTION
**Pull request by aditya3kumar**

### CODE REVIEW

File: database/connection.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import pymysql
+
+#database connection
+connection = pymysql.connect(host="localhost", user="root", passwd="", database="test")
```
Comment (SudeepK20) [inline database/connection.py:4]: Write this code inside a function.

File: database/connection.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+select_tbl= "select * from tbl_employee "
+cursor.execute(select_tbl)
+
+
+
+#commiting the connection then closing it.
+connection.commit()
+connection.close()
+print("end");
+print("hello")
```
Comment (SudeepK20) [inline database/connection.py:15]: Don't write two print statements. It can be done in one print statement.

File: database/connection.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import pymysql
+
+#database connection
+connection = pymysql.connect(host="localhost", user="root", passwd="", database="test")
+cursor = connection.cursor()
+select_tbl= "select * from tbl_employee "
+cursor.execute(select_tbl)
+
+
```
Comment (SudeepK20) [inline database/connection.py:9]: Remove extra spaces

**Review by SudeepK20 — COMMENTED** [review]
Please address my comments

File: database/connection.py (hunk 2 above)
Comment (aditya3kumar) [reply database/connection.py:15]: It can be done but we need to print both thing separately

Comment (SudeepK20) [reply database/connection.py:15]: Can you please explain to me the reason?

**Pull request merged**